### PR TITLE
All in one build - follow ups

### PR DESCRIPTION
**Pull request by hjanetzek**
- Fix segv when loading geojson data. (Tags constructor did not initialize Properties shared_ptr)

- On linux make sure to build with CXX11_ABI for newer gcc to enable SSO strings,
  use libc++ when using clang (on linux it defaults to libstdc++, so this should make it better more comparable with the android build)

### CODE REVIEW

**Comment by hjanetzek** [comment]
merging soon - if no one objects

**Comment by tallytalwar** [comment]
go go go.

**Comment by karimnaaji** [comment]
I noticed some small things that might be good to address in this PR, `make tests` is not available anymore, also headers not showing up in xcode project.

**Comment by tallytalwar** [comment]
Yup headers have not been showing up in xcode for a while now. (I rarely use xcode these days so forgot about it).

**Comment by hjanetzek** [comment]
@karimnaaji could you try the source_path approach? It's hard to test without xcode :)

Maybe this macro could be adapted and added to utils.cmake
http://cmake.3232098.n2.nabble.com/cmake-xcode-c-header-files-td7020444.html

**Comment by karimnaaji** [comment]
Yep I can look into that!

File: core/src/scene/styleParam.cpp [hunk 1]
```diff
@@ -159,7 +160,7 @@ StyleParam::Value StyleParam::parseString(StyleParamKey key, const std::string&
     case StyleParamKey::style:
         return _value;
     case StyleParamKey::font_size: {
-        float fontSize;
+        float fontSize = 10.f;
```
Comment (karimnaaji) [inline core/src/scene/styleParam.cpp]: I remember that discussing with @tallytalwar to remove those here to keep the default style value parameters in the struct they belong to (just to look up in the same place to edit the defaults). Is there any other reason to add them here?

Comment (hjanetzek) [inline core/src/scene/styleParam.cpp]: This should just makes sure that fontSize is not used with an arbitrary (undefined) value

Comment (karimnaaji) [inline core/src/scene/styleParam.cpp]: Yeah anyway a warning if that value is used a warning would be generated so 0.f would be good I guess.

**Comment by karimnaaji** [comment]
Good to go!

**Pull request merged**